hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/KeyManagerImpl.java

```diff
@@ -893,17 +893,18 @@ private OmMultipartInfo createMultipartInfo(OmKeyArgs keyArgs,
       // Not checking if there is an already key for this in the keyTable, as
       // during final complete multipart upload we take care of this.
 
-
+      long currentTime = Time.now();
       Map<Integer, PartKeyInfo> partKeyInfoMap = new HashMap<>();
       OmMultipartKeyInfo multipartKeyInfo = new OmMultipartKeyInfo(
-          multipartUploadID, partKeyInfoMap);
+          multipartUploadID, currentTime, keyArgs.getType(),
```

Comment:
This change needs to be done here.https://github.com/bharatviswa504/hadoop-ozone/blob/master/hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/s3/multipart/S3InitiateMultipartUploadRequest.java#L146

Reply:
For write requests old code is not used anymore.

Reply:
Hi @bharatviswa504, I had already changed this in my previous patch.